var/spack/repos/builtin/packages/neurodamus-asciitoh/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+from spack import *
+import shutil
+import os
+
+class NeurodamusAsciitoh(Package):
+    """Neurodamus Library necessary to convert from ASCII to H5"""
+   
+    homepage = "ssh://bbpcode.epfl.ch/sim/utils/asciitoh5"
+    git      = "ssh://bbpcode.epfl.ch/sim/utils/asciitoh5"
+
+    version('develop', git=git, branch='master')
+    version('0.1', git=git, tag='0.1')
+
+    depends_on('neuron~binary+python~mpi')
```

Comment:
why `~binary+python`? Is python required for the scripts?

Reply:
I just used the same dependency of `neurodamus`. If you think `python` is useless I can remove it.

Reply:
> If you think python is useless I can remove it.
I would try `neuron~mpi`. If builds, there is no reason to add more constraints.